### DISCUSSION TITLE
Video to explain how to install crunchy-cli.

### DISCUSSION THREAD

**Discussion by tatara-blue** [crunchy-labs/crunchy-cli#374] [General]
Hi.

I would like to make a video on YouTube to explain how to install this on Windows.
Do you allow to do so?

**Comment by bytedream** [comment]
Sure! This would be helpful, the question how to install/use it on windows pops up every now and then

**Comment by ySergi** [comment]
You don't "install" this, this is a CLI program, wich means you just have to run it from the terminal.
